src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -30,12 +30,12 @@
         <Style x:Key="GlyphPathStyle"
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
+                            Value="{StaticResource {x:Static SystemColors.GrayTextBrushKey}}" />
```

Comment:
I think you might want to use one of the control* colors. GrayText is usually for disabled UI elements.